components/push/android/build.gradle

```diff
@@ -133,7 +132,7 @@ afterEvaluate {
 
 apply from: "$rootDir/publish.gradle"
 ext.configurePublish(
-        'org.mozilla.push',
+        'mozilla.appservices.push',
```

Comment:
Is this right? I thought we wanted to be using org.mozilla.appservices for this for everything.

Reply:
We're going by this recent PR: https://github.com/mozilla/application-services/pull/709